sysid-application/src/main/native/cpp/view/Analyzer.cpp

```diff
@@ -498,6 +460,8 @@ void Analyzer::SelectFile() {
       m_manager =
           std::make_unique<AnalysisManager>(m_location, m_settings, m_logger);
       m_type = m_manager->GetAnalysisType();
+      m_factor = m_manager->GetFactor();
+      m_unit = m_manager->GetUnit();
```

Comment:
Just noting that this fixes a bug where units per rotation wasn't initialized if the JSON failed to load, so it contained garbage.